content/en/profiler/compare_profiles.md

```diff
@@ -2,7 +2,7 @@
 title: Compare Profiles
 kind: documentation
 further_reading:
-    - link: 'tracing/profiler/enabling'
+    - link: 'profiler/enabling'
       tag: 'Documentation'
       text: 'Enable continuous profiler for your application'
     - link: 'getting_started/profiler'
```

Comment:
The preview link is not generating for these two Blog links due to indentation issues.

Reply:
Also please remove the period after "Comparison" in Line 16.

Reply:
Nice catch, it's actually the use of `tags:` instead of `tag:` that is causing that. Fixed